apps/demos/Demos/Scheduler/ToolbarCustomization/Angular/app/app.component.ts

```
@@ -0,0 +1,98 @@
+import { NgModule, Component, ViewChild, enableProdMode } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import {
+  DxSchedulerModule,
+  DxSchedulerComponent,
+  DxTemplateModule,
+  DxButtonModule,
+} from 'devextreme-angular';
+import DataSource from 'devextreme/data/data_source';
+import type { DxButtonTypes } from 'devextreme-angular/ui/button';
+import { DxSelectBoxModule, type DxSelectBoxTypes } from 'devextreme-angular/ui/select-box';
+import {
+  Service, Assignee
+} from './app.service';
+import {schedulerDataSource} from "../../React/data";
```

Btw, how this line pass the ```lint``` check o_O
The prettier should throw an error (comma styles)